pocs/state/states/default/pointing.py

```diff
@@ -18,6 +16,12 @@ def on_enter(event_data):
 
     pocs.next_state = 'parking'
 
+    # Get pointing parameters
+    pointing_config = pocs.config['pointing']
+    num_pointing_images = pointing_config.get('max_iterations', 3)
+    should_correct = pointing_config.get('auto_correct', False)
```

Comment:
Seems like False is the wrong default.

Reply:
In the end I might change the default in the config to be False, it sort of depends on what we find out from iOptron about usage of the command. I kept False as default here until we confirm as it's easier to change in the config.

Reply:
My point was that, IIUC, the old behavior was equivalent to should_correct==True, so the default should be True.

Reply:
The very old behavior was equal to that but that has not how we have been operating for about 2 years.

To be clear, we always do a pointing image at the beginning of the observation sequence. This option is making it do a "sync" after the initial pointing image, reslew to intended coordinates, then perform another pointing image. It can do this in a loop (`max_iterations`) until it reaches the `threshold`. Usually that's just one additional pointing image.